### ISSUE TITLE
CircleCI fails - repeated flakiness

### ISSUE THREAD

**Issue by ronnno** [orbs-network/orbs-network-go#391]
https://circleci.com/workflow-run/56bdbb6f-aa2e-45f4-9575-3dc14a874d07
https://circleci.com/workflow-run/aafabc82-eee1-4e85-a930-bc4b39fa0771

FAIL: TestNonLeaderPropagatesTransactionsToLeader (10.10s)

and 

FAIL: TestGammaCliDeployWithUserDefinedContract (9.72s)

**Comment by talkol** [comment]
```
--- FAIL: TestGammaCliDeployWithUserDefinedContract (9.72s)
	require.go:201: 
			Error Trace:	gamma_server_and_cli_test.go:220
			            				gamma_server_and_cli_test.go:282
			Error:      	Not equal: 
			            	expected: 0xbbc
			            	actual  : 0x0
			Test:       	TestGammaCliDeployWithUserDefinedContract
```

**Comment by talkol** [comment]
```
orbs-acceptance_1  | --- FAIL: TestNonLeaderPropagatesTransactionsToLeader (10.10s)
orbs-acceptance_1  | 	simple_transfer_test.go:46: testing network with 2 nodes running CONSENSUS_ALGO_TYPE_BENCHMARK_CONSENSUS
orbs-acceptance_1  | panic: statePersistence.WaitUntilCommittedBlockOfHeight failed: timed out waiting for block at height 8 [recovered]
orbs-acceptance_1  | 	panic: statePersistence.WaitUntilCommittedBlockOfHeight failed: timed out waiting for block at height 8
orbs-acceptance_1  | 
```

This looks like the issue @ronnno looked at where state storage went out of sync with block storage

**Comment by ronnno** [comment]
Closing,
Issue with TestNonLeaderPropagatesTransactionsToLeader was both introduced and fixed in PR #366
Opened #400 for TestGammaCliDeployWithUserDefinedContract
